modules/nf-core/longphase/haplotag/main.nf

```diff
@@ -0,0 +1,66 @@
+process LONGPHASE_HAPLOTAG {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/longphase:1.7.3--hf5e1c6e_0':
+        'biocontainers/longphase:1.7.3--hf5e1c6e_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai), path(snps), path(svs), path(mods)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+
+
+    output:
+    tuple val(meta), path("*.bam") , emit: bam , optional: true
+    tuple val(meta), path("*.cram"), emit: cram, optional: true
```

Comment:
These should be the same output channel as of the new guidelines (currently still a PR, but will be in there soon: https://github.com/nf-core/website/pull/2661)

Reply:
Nice! What should we call the bam/cram output channel, bam?